### ISSUE TITLE
Table of contents (toc) does not show 

### ISSUE THREAD

**Issue by enryH** [JanPalasek/pretty-jupyter#75]
I tried you example notebook with interactive components. If I import `itables`, then the toc is not extended/visable. Any idea why this happens?

If I test it with [your interactive example notebook](https://github.com/JanPalasek/pretty-jupyter-examples/blob/main/interactive-components/interactive-components.ipynb), it works when I do not re-execute the notebook, but it fails when I do re-execute it. Any idea which other package could cause the trouble?

```bash
# Name                    Version                   Build  Channel
itables                   1.7.0                    pypi_0    pypi
pretty-jupyter            2.0.5                    pypi_0    pypi
```

**Comment by JanPalasek** [comment]
Hello, thanks for reporting the issue. Ill look at it asap.

**Comment by JanPalasek** [comment]
@enryH If you can, I suggest to downgrade itables to 1.6.4 for now. This problem is in long-dead jquery plugin tocify and it's gonna be hard to fix.

**Comment by enryH** [comment]
Hey! I can:) And I tried it - works as intended. I am unfortunately not a JS programmer (although I had contact to TypeScript...). I hope there is a way to replace `[tocify](https://github.com/gfranko/jquery.tocify.js) in the future...

I tried to see what change between version `1.6.4` and `1.7.0`, but the [tags](https://github.com/JanPalasek/pretty-jupyter/releases) and pip release do not correspond somehow...

**Comment by JanPalasek** [comment]
Glad it worked!

Maybe tempering with the following few lines will fix the issue:
https://github.com/JanPalasek/pretty-jupyter/blob/c104ae0e026139a3acad9b249c3d86b599efa689/src/pretty_jupyter/templates/pj/index.html.j2#L152
I will look into that.

**Comment by JanPalasek** [comment]
The problem is that itables creates loads its own jQuery and overrides the jQuery that tocify uses. I used to be able to detect that and fix it with the specified code, but after the new version the detection does not seem to work.

**Comment by JanPalasek** [comment]
Fixed TOC for itables of version 1.7.0. Ain't pretty, but it will work.
